Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -524,6 +524,21 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Returns a format localized string.
+    ///
+    ///    "%d Swift %d Objective-C".formatLocalized(1, 2) -> 1 Swift 2 Objectiv-C
+    ///
+    /// - Parameters:
+    ///   - comment: Optional comment for translators.
+    ///   - arguments: Arguments used by format.
+    /// - Returns: Format localized string.
+    func formatLocalized(comment: String = "", _ arguments: CVarArg...) -> String {
```

Comment:
You could just update the `localized` function to include the `_ arguments:` parameter, as it won't break backwards compatibility and would just add functionality there.

Reply:
function name clearly explains what it does. let's keep it as is.